rma_delivery/migrations/14.0.1.1.0/pre-migration.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 ForgeFlow S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if openupgrade.column_exists(env.cr, "delivery_carrier", "default_carrier_id"):
+        openupgrade.rename_fields(
+            env,
+            [
+                (
+                    "delivery.carrier",
+                    "delivery_carrier",
```

Comment:
This should be the model and table of the field being renamed, in this case `rma.operation`